monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+from copy import deepcopy
+from glob import glob
+from typing import Sequence, Union
+
+from monai.auto3dseg.algo_gen import Algo, AlgoGen
+from monai.bundle.config_parser import ConfigParser
+from monai.utils import ensure_tuple
+
+__all__ = ["BundleAlgo", "BundleGen", "SegresnetAlgo", "DintsAlgo"]
+
+
+class BundleAlgo(Algo):
+    """
+    An algorithm represented by a set of bundle configurations and scripts.
+
+    ``BundleAlgo.cfg`` is a ``monai.bundle.ConfigParser`` instance.
+
+    ..code-block:: python
+
+        import glob
+
+        from monai.apps.auto3dseg import BundleAlgo
+
+        algo = BundleAlgo(
```

Comment:
For a five-fold workflow, is the BundleAlgo called 5 times?

Reply:
The iteration in the `generate` of `BundleAlgo` solved my question. The comment is closed.